rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] The `book` and `person` should also be in attr_accessor since they'll be used latter on your code